server/app/controllers/login.py

```diff
@@ -0,0 +1,109 @@
+from flask import render_template, abort, request, flash, redirect, url_for
+from app import app
+from flask_wtf import FlaskForm
+from flask_login import current_user, login_user, logout_user, UserMixin, LoginManager
+
+from wtforms import StringField, PasswordField, BooleanField, SubmitField
+from wtforms.validators import InputRequired, Email, Length
+from werkzeug.urls import url_parse
+from werkzeug.security import generate_password_hash, check_password_hash
+
+from app import app
+from app.repository import Repository, UserRole
+
+class LoginForm(FlaskForm):
+    username = StringField("Your login name", validators=[InputRequired(), Length(min=3, max=20)])
+    password = PasswordField("Your password", validators=[InputRequired(), Length(min=5, max=80)])
+    remember = BooleanField("Remember me")
+
+    submit = SubmitField("Sign In")
+
+class SatnogsUser(UserMixin):
```

Comment:
We plan to abandon "Satnogs" name for out project. Maybe "ApplicationUser" will be better?

Reply:
That name is a too long IMHO, but I don't care, so renamed as suggested.